src/private.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <pthread.h>
+#include <rtl-sdr.h>
+#include <stdio.h>
+
+#include <nrsc5.h>
+
+#include "config.h"
+#include "defines.h"
+#include "input.h"
+#include "output.h"
+
+struct nrsc5_t
+{
+    rtlsdr_dev_t *dev;
+    FILE *iq_file;
+    uint8_t samples_buf[128 * 256];
+    float freq;
+    int decimation;
```

Comment:
This seems unused. `nrsc5_open_iq` sets it to 2, and that's it.

Reply:
Yes, left over from the SoapySDR support.